src/main/java/core/basesyntax/controller/UserHandler.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.Dao;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class UserHandler implements ConsoleHandler {
```

Comment:
what if name will contain smth else than letters?

Reply:
For now I don't know, what if someone would like to use a nickname instead of real name

Reply:
ok. true. and one more thing: i didn't see tests for your app, and your app doesn't do anything. does it really work?))

Reply:
It isn't necessary to write tests)
Yes, it does work

Reply:
:+1:

Reply:
Added a few lines to Main